transformations/diacritic_removal/transformation.py

```diff
@@ -0,0 +1,22 @@
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+import unicodedata
+
+class DiacriticRemoval(SentenceOperation):
+
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
```

Comment:
Please add the `keywords` variable (see email from NL-Augmenter organizer).

Reply:
Thanks, I have added the keywords now.